src/components/NavWizard/NavWizard.tsx

```diff
@@ -117,7 +117,8 @@ const NavWizard = (props: NavWizardProps) => {
           <Button
             ref={backBtnRef}
             type="button"
-            className="bsds-button-nav-back"
+            className="bsds-button-text"
+            icon="chevronLeft"
```

Comment:
icon is a little smaller in the preview (16x16) than in main (18x18). can we confirm with catie that she's good with the change?

https://main--64e769384ef6b440f819fcec.chromatic.com/?path=/story/components-wizard-navigation--playground
https://64e769384ef6b440f819fcec-wnqvjfhjcn.chromatic.com/?path=/story/components-wizard-navigation--playground

Reply:
Catie said it's ok!